target/linux/ramips/base-files/lib/ramips.sh

```diff
@@ -235,7 +235,7 @@ ramips_board_detect() {
 	*"HC5962")
 		name="hc5962"
 		;;
-	*"HG255D")
```

Comment:
Drop this change. It should make no difference since the "HuaWei" is already covered by the wildcard "*".